hack/stabilization-changes.py

```diff
@@ -94,24 +94,37 @@ def stabilize_channel(name, channel, channels, channel_paths, **kwargs):
             **kwargs)
 
 
+def human_readable_relativedelta(delay):
+    delta = {}
```

Comment:
your later `keys()` iteration seems like you want to use [`OrderedDict()`][1] here.

[1]: https://docs.python.org/3/library/collections.html#collections.OrderedDict